### PR TITLE
Test2

### PR DESCRIPTION
**Pull request by Matt-Warnock**
All finished with better practice css

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -1,9 +1,83 @@
+
 h1 {
   display: flex;
   justify-content: center;
```
Comment (octopusinvitro) [inline styles.css]: In this case you can replace these two lines:

```css
  display: flex;
  justify-content: center;
```
with just
```css
text-align: center;
```

The reason is that `h1` is a block element, so it occupies the full width available by default. So `text-align` will center its contents.
You would use flexbox if you wanted to center boxes with a width, or when you have several boxes in the same line.

File: randomizer.js [hunk 2]
```diff
@@ -3,3 +3,13 @@ var myRandomNumber = Math.floor(Math.random() * 100);
...
 console.log(mySentence);
+var myButton = document.getElementById('myButton');
+myButton.addEventListener('click', function(event){
+  var myNoun = document.getElementById('noun').value;
+  var myVerb = document.getElementById('verb').value;
+  var myAdject = document.getElementById('adject').value;
+  var myAdverb = document.getElementById('adverb').value;
+  var myMadLab = `Do you ${myVerb} your ${myAdject} ${myNoun} ${myAdverb}?`;
+  var myTarget = document.getElementById('target');
+  myTarget.innerHTML = myMadLab;
```
[the remark this answers is not in the document]
Comment (Matt-Warnock) [reply randomizer.js]: good suggestion. I've shorten all the var names now

File: styles.css (hunk 1 above)
Comment (Matt-Warnock) [reply styles.css]: Thanks for the tip. I've done that change, makes it nice a clean :)

File: JS challenge1.html [hunk 3]
```diff
@@ -1,13 +1,37 @@
...
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="styles.css">
+  <link href="https://fonts.googleapis.com/css?family=Orbitron&display=swap" rel="stylesheet">
+  <title>Age Randomizer</title>
+</head>
+
+<body>
+  <div class="mad_lab">
+    <h1>Mad Lab Generater</h1>
+    <div class="noun">
```
[the remark this answers is not in the document]
Comment (Matt-Warnock) [reply JS challenge1.html]: Thank you :)

**Comment by octopusinvitro** [comment]
:+1: good to go! :shipit:

**Comment by Matt-Warnock** [comment]
Mad lab JavaScript and some styling done

**Pull request merged**